pkgs/applications/misc/plots/default.nix

```diff
@@ -0,0 +1,88 @@
+{ fetchFromGitHub
+, fetchzip
+, gobject-introspection
+, lib
+, libadwaita
+, python3Packages
+, python3
+, wrapGAppsHook4
+, fetchPypi
+}:
+let
+  pyglm = python3Packages.buildPythonPackage rec {
+    pname = "PyGLM";
+    version = "2.5.7";
+    src = fetchPypi {
+      inherit pname version;
+      hash = "sha256-OP3/hOZBbez/4cWHRjKAfVi5jFUIzKNXm7L1My4Jn9g=";
+    };
+    doCheck = false;
+  };
+  latinmodern-math = fetchzip {
+    url = "https://www.gust.org.pl/projects/e-foundry/lm-math/download/latinmodern-math-1959.zip";
+    hash = "sha256-et/WMhfZZYgP0S7ZmI6MZK5owv9bSoMBXFX6yGSng5Y=";
+  };
+in
+python3Packages.buildPythonApplication rec {
+  pname = "plots";
+  version = "0.8.5";
+
+  src = fetchFromGitHub {
+    owner = "alexhuntley";
+    repo = "Plots";
+    rev = "v${version}";
+    hash = "sha256-GjNpaorxkkhZsqrKq4kO5nqF5+4I4tmSc023AZpY8Sw=";
+  };
+
+  propagatedNativeBuildInputs = [
+    gobject-introspection
+    wrapGAppsHook4
+  ];
+  propagatedBuildInputs = [
+    libadwaita
+    pyglm
+    (python3.withPackages (p: with p; [
+      numpy
+      pygobject3
+      lark
+      jinja2
+      freetype-py
+      pyopengl
+      pycairo
+    ]))
+  ];
+
+  nativeCheckInputs = [
+    (python3.withPackages (p: with p; [
+      pytest
+    ]))
+  ];
+
+  postInstall = ''
+    install -D ${latinmodern-math}/otf/latinmodern-math.otf -t $out/share/fonts/
+
+    install -D res/com.github.alexhuntley.Plots.metainfo.xml -t $out/share/metainfo/
+    install -D res/com.github.alexhuntley.Plots.desktop -t $out/share/applications/
+    install -D res/com.github.alexhuntley.Plots.svg -t $out/share/icons/hicolor/scalable/apps/
+    install -D res/com.github.alexhuntley.Plots-symbolic.svg -t $out/share/icons/hicolor/symbolic/apps/
+
+    for lang_dir in help/*; do
+      lang=$(basename "$lang_dir")
+      install -D -t $out/share/help/$lang/plots/ $lang_dir/*
+    done
+  '';
+
+  meta = with lib; {
+    description = "Plots is a graph plotting app for GNOME written on python.";
```

Comment:
We generally don't repeat the program name in description, and don't put a dot at the end. I think I'd write it as `Graph plotting app for GNOME`. (That it's written in python isn't that relevant, I think, but if you want to keep it I belive it should be `written in [...]`, not `written on [...]`.)

Reply:
Fixed, thank you